src/Utilities/tests/test_ModernStringUtils.cpp

```diff
@@ -31,4 +31,61 @@ TEST_CASE("ModernStringUtils_strToLower", "[utilities]")
         "\xab"
         "not_just_ascii");
 }
+
+TEST_CASE("ModernStringUtils_split", "[utilities]")
```

Comment:
There should be some tests for edge cases - empty string, string with no delimiters, string with empty delimiter, string that is only delimiters.